core/cmd/hoverfly/main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 		requestCache = nil
 	}
 
+	cfg.DisableProxyAuthoriation = *disableProxyAuth
```

Comment:
But why not accept both authorization header? Is it not better to follow Postel's Law and accept both values and then disable the insecure option when configuring Hoverfly for a secure deployment?

I don't think having both would be confusing. We change the docs to use the `X-HOVERFLY-AUTHORIZATION` header but mention that by default, we still accept the `Proxy-Authorization` for those using Hoverfly locally and want to use curl.
```
curl --proxy user:pass@localhost:8888 http://hoverfly.io
```